Assets/Scripts/EncounterScripts/TileEncounterTrigger.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TileEncounterTrigger : MonoBehaviour
+{
+	public Encounter encounter;
+	private EncounterManager encounterManager;
```

Comment:
make encounter private and serialize it for the inspector, also follow variable naming conventions:
private -> _variable
public -> Variable